newton/tests/test_schema_resolver.py

```
@@ -0,0 +1,863 @@
+from newton import ModelBuilder
+from newton._src.sim.model import AttributeFrequency
+from newton._src.utils.import_usd import parse_usd
+from newton._src.utils.schema_resolver import MjcPlugin, NewtonPlugin, PhysxPlugin, Resolver
+from newton.tests.unittest_utils import USD_AVAILABLE
+
+if USD_AVAILABLE:
+    try:
+        from pxr import Usd as _Usd
+
+        Usd: Any = _Usd
+    except Exception:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)